python/vowpalwabbit/pyvw.py

```diff
@@ -1215,6 +1210,70 @@ def __str__(self):
         )
 
 
+class SlatesLabelType(IntEnum):
+    UNSET = pylibvw.vw.tUNSET
```

Comment:
I like this. We should do that for the other enums too

Reply:
Yea agreed